### DISCUSSION TITLE
后续是否考虑支持数据导出为csv呢？

### DISCUSSION THREAD

**Discussion by DustinMaple** [focus-creative-games/luban#108] [General]
首先，我并不否定bin和json，他们都有各自的应用场景。不过csv也是具有一定优势的。
1. 与json相比，其数据文件要更小，因为除了前几行的表头，剩下的都是数据了。不像json，每条数据都带有字段名。
2. 从数据格式上，更接近excel，并且能用excel打开。像目前项目里，策划在查看更新记录的时候，是去看csv的变更记录的，毕竟excel无法进行内容对比。

**Comment by foreversos** [comment]
官方都没答复啊`
